components/raftstore/src/store/fsm/peer.rs

```diff
@@ -2830,11 +2842,24 @@ where
         self.fsm.hibernate_state.count_vote(from.get_id());
     }
 
-    fn on_availability_response(&mut self, from: &metapb::Peer, msg: &ExtraMessage) {
+    fn on_availability_response(&mut self, msg: RaftMessage) {
```

Comment:
As it's only used by witness, I don't suggest to change it

Reply:
Yep, that's right. It was reverted.

Reply:
And the abnormal peer will be cleared by the responding message with the `RaftMessage::is_tombstone` == true.